src/marketing/product-compare.md

```diff
@@ -27,3 +31,45 @@ Unlike the [Recently Viewed / Compared Products ]({% link marketing/products-vie
 
     ![Example storefront - compare products]({% link marketing/assets/storefront-compare-products.png %}){: .zoom}
     _Compare Products_
+
+From the notification message after product was added to comparison list:
+
+1. The customer finds the products that they want to compare and clicks the **Add to Compare** link for each.
+
+1. In the appeared top message notification, clicks _comparison list_ link.
+
+   ![Compare Products Notification]({% link catalog/assets/notification-comparison-list.png %}){: .zoom}
+   _Compare Products Notification_
+
+Customer is redirected to comparison list and can do additional actions.
+
+From the **Compare Products** block in the header:
+
+1. The customer finds the products that they want to compare and clicks the **Add to Compare** link for each.
+
+1. In the header, near the search field clicks _Compare Products_ link.
+
+   ![Compare Products Header]({% link catalog/assets/compare-products-header.png %}){: .zoom}
+   _Compare Products Header_
+
+From the **My Account** dashboard:
+
+1. The customer adds needed products into comparison list.
+
+1. Navigates to **My Account**.
+
+1. Finds **Compare Products** block and clicks <span class="btn">Compare</span>.
+
+   ![Customer's Account Dashboard Compare Products Block]({% link customers/assets/my-account-compare-block.png %}){: .zoom}
+   _Customer's Account Dashboard Compare Products Block_
+
+## Additional actions in comparison list
+
+(![Delete icon]({% link assets/icon-delete-x.png %})) - deletes a single item from comparison list.
+
+<span class="btn">Add to Cart</span> - adds product into shopping cart.
+If the product has any configurations, customer redirects to product page, selects configurable options, and then clicks <span class="btn">Add to Cart</span>.
+
+_Wishlist icon_ - adds product into wishlist, in case the wishlist functionality is enabled in configurations.
+
+_Print This Page_ - prints comparison list page.
```

Comment:
We usually do a table for these items

```suggestion
(![Delete icon]({% link assets/icon-delete-x.png %})) | Deletes a single item from comparison list.
<span class="btn">Add to Cart</span> | Adds product into shopping cart. If the product has any configurations, the page redirects the customer to the product page where they select the configurable options and then click <span class="btn">Add to Cart</span>.
_Wishlist icon_ | Adds product into wishlist, in case the wishlist functionality is enabled in configurations.
_Print This Page_ | Prints comparison list page.
```